helix-core/src/syntax.rs

```diff
@@ -537,6 +537,10 @@ impl Loader {
         None
     }
 
+    pub fn language_configs(&self) -> impl Iterator<Item = &Arc<LanguageConfiguration>> {
```

Comment:
I think creating a function like `language_config_for_scope` would be more homogenous with the code.

Reply:
Yes. :slightly_smiling_face:  Though, now that you mentioned it, I could've added a `language_config_for_id` functions which could've been useful in the actual command implementation...